Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/ApplicationComponent.kt

```diff
@@ -125,6 +128,8 @@ interface ApplicationComponent : AndroidInjector<CoronaWarnApplication> {
 
     @AppScope val appScope: AppCoroutineScope
 
+    val initializers: Set<@JvmSuppressWildcards Initializer>
```

Comment:
Why is this needed?

Reply:
this is leftover , I was calling it from 
```
AppInjector.init(this).let { compPreview ->}
```